integration/package-operator/objectset_test.go

```diff
@@ -418,6 +418,15 @@ func defaultObjectSetRev2(cm1, cm3 *corev1.ConfigMap, rev1 *corev1alpha1.ObjectS
 	}, nil
 }
 
+func isControllerOf(objectReferences []corev1alpha1.ControlledObjectReference, expected corev1alpha1.ControlledObjectReference) bool {
```

Comment:
I could have sworn we had this method flying around somewhere but I can't find it anymore. Would it be a good idea to move this one here to some util pkg?